### PR TITLE
add a previous dataset property

### PR DESCRIPTION
**Pull request by IsaackMwenda**
This will be updated once a message is moved from its original dataset through ws correction.

### CODE REVIEW

File: engagement_database/data_models.py [hunk 1]
```diff
@@ -50,6 +50,8 @@ def __init__(self, text, timestamp, participant_uuid, direction, channel_operato
         :type coda_id: str | None
         :param last_updated: Timestamp this message was last updated in Firestore, or None if it does not yet exist.
         :type last_updated: datetime.datetime | None
+        :param previous_datasets: Datasets which this message originally belonged to/moved from. If None, a return an empty list
```
Comment (as2388) [inline engagement_database/data_models.py:53]:
```suggestion
        :param previous_datasets: Datasets which this message originally belonged to/moved from. If None, initialises with an empty list
```

File: engagement_database/data_models.py [hunk 2]
```diff
@@ -24,7 +24,7 @@ class MessageDirections(object):
 
 class Message(object):
     def __init__(self, text, timestamp, participant_uuid, direction, channel_operator, status, dataset, labels,
-                 message_id=None, coda_id=None, last_updated=None):
+                message_id=None, coda_id=None, last_updated=None, previous_datasets=None,):
```
Comment (as2388) [inline engagement_database/data_models.py:27]:
```suggestion
                 message_id=None, coda_id=None, last_updated=None, previous_datasets=None,):
```

File: engagement_database/data_models.py (hunk 1 above)
Comment (as2388) [reply engagement_database/data_models.py:53]: I think this is the right thing to do, but I don't see a if previous_datasets is None: previous_datasets = [] in the constructor

**Review by as2388 — APPROVED** [review]

File: engagement_database/data_models.py (hunk 1 above)
Comment (IsaackMwenda) [reply engagement_database/data_models.py:53]: Was abit conflicted here whether we should return None or an empty list. I think unlike labels previous_datasets should be None unless the message moves. So have created a list here https://github.com/AfricasVoices/Test-Pipeline-Engagement-DB/pull/17/files#diff-834f785e8021c007d4aed1e4aa8bdd5c9deafba3f3687e41bcc6ce0f9b124327R101 only when the message moves. If you agree will update the doc string accordingly.

**Pull request merged**